pytorch_lightning/trainer/trainer.py

```diff
@@ -521,7 +522,7 @@ def _call_and_handle_interrupt(self, trainer_fn: Callable, *args: Any, **kwargs:
             # reset bookkeeping
             self.state.stage = None
             self.on_exception(exception)
-            raise
+            raise exception
```

Comment:
This change is not necessary, is it?

Reply:
no it's not, I thought it was clearer this way but I can put it back

Reply:
Yes please :)

https://stackoverflow.com/a/45986945